tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -1,45 +1,49 @@
 *** Settings ***
-Resource         ../../Resources/ODS.robot
-Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
-Test Setup      Dashboard Test Setup
-Test Teardown   Dashboard Test Teardown
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+
+Test Setup          Dashboard Test Setup
+Test Teardown       Dashboard Test Teardown
 
 
 *** Test Cases ***
 Verify Resource Link Http status code
```

Comment:
I don't know if you've run robocop on this, but we should keeps TC and keyword names' formatting consistent. Right now we are enforcing `Title Case`, so the first letter of each work should be capitalized:
`Verify Resource Link Http Status Code`
I would also use `HTTP` since that is usually capitalized, being an acronym.

Same applies to other TC names in the suite.